### ISSUE TITLE
Failing to find playground or handler package

### ISSUE THREAD

**Issue by ajoseps** [99designs/gqlgen#961]
### What happened?
I attempted to build my project with the following imports:
github.com/99designs/gqlgen
github.com/99designs/gqlgen/graphql/handler
github.com/99designs/gqlgen/graphql/playground

Received the following error upon running `go build`:

`cannot load github.com/99designs/gqlgen/graphql/handler: module github.com/99designs/gqlgen@latest found (v0.10.2), but does not contain package github.com/99designs/gqlgen/graphql/handler`

### What did you expect?
The packages to be found

### Minimal graphql.schema and models to reproduce
N/A
### versions
 - `gqlgen version`? 
`v0.10.2`
 - `go version`?
`go1.13.5 linux/amd64`
 - dep or go modules?
go modules

**Comment by ajoseps** [comment]
I believe changing the import statement to `import {module name}/{package name}`, where module name is `github.com/99designs/gqlgen` resolves the issue. e.g.
**FROM**
`import "github.com/99designs/gqlgen/graphql/handler"`
**TO**
 `import "github.com/99designs/gqlgen/handler"`

**Comment by ajoseps** [comment]
The examples might be out of date

**Comment by ajoseps** [comment]
It actually seems like  `import "github.com/99designs/gqlgen/handler"` does not import the same package as `import "github.com/99designs/gqlgen/graphql/handler"` (which still does not work for me). I cannot find the `New` function

**Comment by ajoseps** [comment]
I was able to replicate this error in go playground which leads me to believe it is not my local setup

**Comment by ajoseps** [comment]
I have also tested with go 1.12.9 and replicated the same thing

**Comment by fwojciec** [comment]
go get is not pulling the latest version of the code from github?
This has fixed the problem for me:

`go get -u github.com/99designs/gqlgen@f869f5a85385745d5854daaa25eab5571b04b245`

(f869f5a85385745d5854daaa25eab5571b04b245 is the latest commit as of this moment)

**Comment by ajoseps** [comment]
@fwojciec thank you for the advice, that seemed to have worked. I'm going to keep this issue open however since the go get should be pulling the latest version I believe.

**Comment by fwojciec** [comment]
I've run into the same :)  Thanks for filing the report -- it was helpful in helping me identify the issue!

In case anyone else runs into this --:

`go get` command basically fetches the last tagged version of the repo by default (which makes sense).  The current tagged version for the gqlgen repo is 0.10.2, while the changed package structure in the examples seems to be work in progress for the 0.11 release (as [described in the docs](https://gqlgen.com/recipes/migration-0.11/)). 

In other words -- as soon as the new version is officially released and tagged everything should work correctly and the workaround above shouldn't be required anymore.

**Comment by vektah** [comment]
Sorry for the confusion, The docs now show the stable version by default with the master docs available at https://gqlgen.com/master/.
